SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -779,6 +780,36 @@ void ASpatialFunctionalTest::DeleteActorsRegisteredForAutoDestroy()
 	}
 }
 
+APlayerController* ASpatialFunctionalTest::GetFlowPlayerController()
+{
+	ASpatialFunctionalTestFlowController* FlowController = GetLocalFlowController();
+	if (!IsValid(FlowController))
+	{
+		return nullptr;
+	}
+	APlayerController* PlayerController = Cast<APlayerController>(FlowController->GetOwner());
+	if (!IsValid(PlayerController))
+	{
+		return nullptr;
+	}
+	return PlayerController;
+}
```

Comment:
This is a small thing, but in tests we should really always have a PlayerController if the flow controller is for a client.
Maybe in this case we could push the error checking into this function, so if the flow controller is not valid, we could do an ensure or even check, or an error log and return nullptr.